sqlmesh/core/loader.py

```diff
@@ -183,9 +183,13 @@ def _load_external_models(self) -> UniqueKeyDict[str, Model]:
         models: UniqueKeyDict[str, Model] = UniqueKeyDict("models")
         for context_path, config in self._context.configs.items():
             schema_path = Path(context_path / c.SCHEMA_YAML)
+            external_models_default_path = context_path / c.EXTERNAL_MODELS_DEFAULT_YAML
             external_models_path = context_path / c.EXTERNAL_MODELS
 
             paths_to_load = []
+            if external_models_default_path.exists():
```

Comment:
Should we just load one or other rather than both? In other words:
```
if external_models_default_path.exists():
    ...
else if schema_path.exists():
    ...
```